charts/consul/test/unit/api-gateway-controller-deployment.bats

```diff
@@ -908,7 +908,7 @@ load _helpers
 #--------------------------------------------------------------------
 # global.cloud
 
-@test "apiGateway/Deployment: fails when global.cloud.enabled is set and global.cloud.secretName is not set" {
+@test "apiGateway/Deployment: fails when global.cloud.enabled is set and global.cloud.clientId.secretName is not set but global.cloud.clientSecret.secretName and global.cloud.resourceId.secretName is set" {
```

Comment:
these `fail` tests will be repeated in all below bats files below, with `server-statefulset.bats` also having a true logic change.

Reply:
Just a suggestion to clarify `true`/`false` vs `set`/`not set`. I feel the former is easier to read in a long line like this.

```suggestion
@test "apiGateway/Deployment: fails when global.cloud.enabled is true and global.cloud.clientId.secretName is false but global.cloud.clientSecret.secretName and global.cloud.resourceId.secretName are both true" {
```

Reply:
```suggestion
@test "apiGateway/Deployment: fails when global.cloud.enabled is true and global.cloud.clientId.secretName is not set but global.cloud.clientSecret.secretName and global.cloud.resourceId.secretName is set" {
```

Reply:
🙏  this has been corrected.